### PR TITLE
Integrating all 59, 60 and 61 reported issues

### PR DESCRIPTION
**Pull request by tulvanchristian**
The commit includes fixes for:
    Support for base64 encoded buffers #59
    Set child name (rootProperty) #60
    Direct access to buffers and accessors #61

### CODE REVIEW

**Comment by bghgary** [comment]
Build is failing..

File: GLTFSDK/Inc/GLTFSDK/BufferBuilder.h [hunk 1]
```diff
@@ -84,6 +84,15 @@ namespace Microsoft
 
             ResourceWriter& GetResourceWriter();
             const ResourceWriter& GetResourceWriter() const;
+            bool setCurrentBufferURI ( const std::string &data )
```
Comment (bghgary) [inline GLTFSDK/Inc/GLTFSDK/BufferBuilder.h:87]: I think the missing semicolon is the main cause of the build break. Also, please use the same spacing style as the rest of the code.

```suggestion
            bool setCurrentBufferURI(const std::string &data);
```

**Review by bghgary — CHANGES_REQUESTED** [review]

File: GLTFSDK/Inc/GLTFSDK/BufferBuilder.h (hunk 1 above)
Comment (tulvanchristian) [reply GLTFSDK/Inc/GLTFSDK/BufferBuilder.h]: Thank you very much for the comments.
I think i fixed the code, alignment and spaces but it seems that the build still fails (on some files I did not modify, especially for the std::pow functions in the Math.h file). Do you have any suggestions for this issue? Or what is the next step ?
I apologize if this is taking longer than usual, but I am new to github procedures.

Comment (bghgary) [reply GLTFSDK/Inc/GLTFSDK/BufferBuilder.h]: I see a bunch of build errors, but they are all in BufferBuilder.h and .cpp. Have you built this locally?

**Comment by bghgary** [comment]
Closing due to inactivity. Please reopen and fix the issues if you still want to merge this change.

**Pull request closed without merging**